src/third-party/hubspot/components/open-chat-button.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from 'react';
+import Icon from '@ably/ui/core/Icon';
+import { identity } from 'lodash';
+
+declare global {
+  interface Window {
+    HubSpotConversations?: {
+      widget: {
+        open: () => void;
+      };
+    };
+    hsConversationsOnReady?: (() => void)[];
+  }
+}
+
+type OpenChatButtonProps = {
+  label: string;
+  labelWhenInactive: string;
+  options: {
+    withIcon: boolean;
+  };
+};
+
+const ACTIVE = true;
+const INACTIVE = false;
+
+const safeWindow = (
+  typeof window === 'undefined' ? { HubSpotConversations: undefined, hsConversationsOnReady: [] } : window
+) as Window;
+
+const OpenChatButton = ({ label, labelWhenInactive, options: { withIcon } }: OpenChatButtonProps) => {
+  const [status, setStatus] = useState(INACTIVE);
```

Comment:
Instead of an ACTIVE and INACTIVE constant, calling this:
`[isActive, setIsActive]` and removing the constants would be better